api-authorizations/src/test/resources/authorizations_api/api_users.yml

```diff
@@ -0,0 +1,110 @@
+description: Get information about user personal UserApi token
+method: GET
+url: /secure/api/user/api/token
+headers:
+  - "Content-Type: application/json"
+response:
+  code: 200
+  content: >-
+    {
+     "action" : "getApiToken",
+     "result" : "success",
+     "data" : {
+       "accounts" : [
+          {
+            "id" : "user1",
+            "name" : "user1",
+            "token" : "",
+            "tokenGenerationDate" : "1970-01-01T00:00:00Z",
+            "kind" : "system",
+            "description" : "number one user",
+            "creationDate" : "1970-01-01T00:00:00Z",
+            "enabled" : true,
+            "expirationDateDefined" : false
+          }
+        ]
+      }
+    }
+---
+description: Update user personal UserApi token
+method: POST
+url: /secure/api/user/api/token
+headers:
+  - "Content-Type: application/json"
+response:
+  code: 200
+  content: >-
+    {
+      "action" : "updateApiToken",
+      "result" : "success",
+      "data" : {
+        "accounts" : [
+          {
+            "id" : "user1",
+            "name" : "user1",
+            "token" : "",
+            "tokenGenerationDate" : "1970-01-01T00:00:00Z",
+            "kind" : "system",
+            "description" : "number one user",
+            "creationDate" : "1970-01-01T00:00:00Z",
+            "enabled" : true,
+            "expirationDateDefined" : false
+          }
+        ]
+      }
+    }
+---
+description: Create user personal UserApi token
+method: PUT
+url: /secure/api/user/api/token
+headers:
+  - "Content-Type: application/json"
+body: >-
+  {
+    "name": "user1",
+    "description": "number one user"
+  }
+response:
+  code: 200
+  content: >-
+    {
+     "action" : "createApiToken",
+     "result" : "success",
+     "data" : {
+       "accounts" : [
+          {
+            "id" : "user1",
+            "name" : "user1",
+            "token" : "generated-test-token",
+            "tokenGenerationDate" : "1970-01-01T01:00:00+01:00",
```

Comment:
try to use different date for different use case, for ex simulating a regenerated token, so that we can see that the correct date ends-up in the correct field.